core/kernel/shared/require.rb

```diff
@@ -467,8 +467,8 @@
     end
 
     ruby_version_is "2.3" do
-      it "complex, enumerator, rational, thread and unicode_normalize are already required" do
-        provided = %w[complex enumerator rational thread unicode_normalize]
+      it "complex, enumerator, rational, and thread are already required" do
```

Comment:
Could you move the new set of provided features in a spec guarding with Ruby 2.5?
```ruby
    ruby_version_is "2.5" do
      it "complex, enumerator, rational, and thread are already required" do
```

Keeping the existing specs ensure old versions don't regress.